{"uuid":"152897d5-77f8-4078-b86a-1a67ba248c42","name":"Scenario Outline: To validate if user getting error message when enter valid python code in operations stack page","children":["a82686ae-0096-44ae-8d08-c8a5e6d2639a"],"befores":[],"afters":[{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055260111,"stop":1709055260111},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055260819,"stop":1709055260819},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055263655,"stop":1709055263656},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055264132,"stop":1709055264133},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055264590,"stop":1709055264590},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055265335,"stop":1709055265336},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055265888,"stop":1709055265889},{"name":"stepDefinitions.ApplicationHooks.tearDown()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055265889,"stop":1709055267090}],"start":1709055256554,"stop":1709055267090}